mscore/scoretab.cpp

```diff
@@ -236,6 +242,17 @@ void ScoreTab::setCurrent(int n)
                   }
             }
 #endif
+#ifdef AVSOMR
+      if (v) {
+
+            Score* score = v->score();
+            if (score->masterScore()->avsOmr()) {
+                        Avs::InfoPopup* info = new Avs::InfoPopup();
```

Comment:
It looks like when you switch from and to the tab with the score being recognized, a new `InfoPopup` gets created but the old one doesn't get deleted. This can be reproduced with the following steps:

1) Import a score with this OMR facility;
2) Repeat a few times:
    - Switch to another tab;
    - Switch to the tab with the score being imported; 
3) Try to close the popup by clicking on it.

Expected result: popup gets closed immediately.
Actual result: you have to click as many times as you have repeated step 2 (plus one more time for the initial popup).

Also this popup doesn't seem to prevent mouse events propagation to `ScoreView` which makes it possible to accidentally make some edits to a score while trying to close the popup (for example, drag some notes or other elements which appeared to be under this popup).

Reply:
thanks, fixed it